CONTRIBUTING.md

```diff
@@ -0,0 +1,106 @@
+# Contributing to DroidconKe-RN App
+
+
+You can contribute to Hey Linda by beta testing, recording meditations, or submitting code.
+If you plan to make a contribution please do so through [our detailed contribution workflow.](#contribution-steps). You can also join us on Slack to discuss ideas.
+
+
+When submitting code, please make every effort to follow existing [conventions and style](https://github.com/droidconKE/droidconKE2022ReactNative/blob/main/STYLEGUIDE.md) in order to keep the code as readable as possible. 
+
+
+
+
+Please note we have a [code of conduct](https://github.com/droidconKE/droidconKE2022ReactNative/blob/main/CODE_OF_CONDUCT.md), follow it in all your interactions with the project.
+
+
+## Submitting a PR
+
+- For every PR there should be an accompanying [issue](https://github.com/droidconKE/droidconKE2022ReactNative/issues) which the PR solves
+
+- The PR itself should only contain code which is the solution for the given issue
+
+- If you are a first time contributor check if there is a [good first issue](https://github.com/droidconKE/droidconKE2022ReactNative/labels/good%20first%20issue) for you
+
+
+<br>
+
+## Contribution steps
+
+
+
+
+1. Fork this repository to your own repositiry. 
+
+2. Clone the forked repositiry to your local machine.
+
+3. Run it locally with our [guide](#how-to-run-locally)
+
+4. Create your feature branch: `git checkout -b my-new-feature`
```

Comment:
I am thinking the feature branch name could be named as
`git checkout -b feature-my-new-feature`